bot.py

```diff
@@ -8,32 +8,48 @@
     data = json.load(json_file)
     TOKEN = data['discordKey']
 
-client = discord.Client()
+client = commands.Bot(command_prefix = '!')
+
+#####################
```

Comment:
I think we need to discuss our coding style because I'm not sure how much we will want these big block comments. Especially because every method has the ```@client.event``` or ```@client.command``` before it.

Reply:
Yes, we should. What sort of standard should we go by? I figured that it would make it easier to read -- sorry if that was a bit silly.